helpdesk_mgmt_partner_sequence/models/helpdesk_ticket.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2020 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class HelpdeskTicket(models.Model):
+    _inherit = "helpdesk.ticket"
+
+    def _prepare_ticket_number(self, values):
+        seq = False
+        if "partner_id" in values or self.partner_id:
+            partner = (
+                self.env["res.partner"].browse(values["partner_id"])
+                if "partner_id" in values
+                else self.partner_id
+            )
+            company_id = (
+                values["company_id"] if "company_id" in values else self.company_id.id
```

Comment:
wouldn't it be better?
company_id = values.get('company_id', self.company_id.id)